rfcs/0001-rfc-process.md

```diff
@@ -0,0 +1,135 @@
+
+* Champion(s): The people making and maintaining the proposal. This can change over time.
+* Stage: Accepted, Deprecated, Implemented
+* Summary
+* Objective (goals and non-goals)
+* Motivation (background)
+* Detailed Design, including these recommended sections:
+    * Implementation Plan
+    * Backward Compatibility
+    * Testing Plan
+    * Performance and Code Size Impact
+    * Interoperability
+    * Security Impact
+    * Documentation Plan
+* Downsides
+* Alternatives
+
+Accepted RFCs are not a guarantee that a feature will be developed and included in Lit. They are an indication that the team has consensus on a feature and design, and is open to accepting PRs to implement it. Things can change, and new information such as implementation feedback may lead the team to deprecate an RFC.
+
+## When to use RFCs
+
+RFCs must be created for new and "substantial" changes. "Substantial" is somewhat subjective, but some clear signs are:
+
+* New packages
+* New modules, classes, directives, etc. in an existing package
+* Most new public-facing API
+* Breaking changes
+
+Changes that do _not_ require an RFC:
+
+* Internal implementation changes
+* Bug fixes
+* Most dev mode changes: warnings, etc.
+
+These criteria leave a category of small changes and experiments open to debate on whether they deserve or require an RFC. We do not wish to slow down development with too much process, so we will leave this area to the discretion of the core team. We will adjust the criteria for requiring RCFs over time.
+
+#### Relationship to Lit Labs
+
+The Lit project also has a labs process for pre-production experimentation and feedback. With the assumption that when we create a labs package we usually intend for it to eventually graduate, most labs packages should have (or be part of) an RFC.
+
+## RFC Process
+
+### Before opening an RFC
+
+RFCs involve more work than an issue, both to open one and to review one. Before opening an RFC is it a good idea to get some indication of interest from the Lit core team or strong support from the community.
+
+You may get this indication in several ways, from informal to formal:
+
+* A feature request issue is closed with a request to open an RFC
+* A core team member directly invites you to open an RFC from a community or support channel like GitHub Discussions, Slack, or Twitter
+* You open a "RRFC" (Requesting Request For Comment) issue in lit/rfcs and get positive feedback 
+
+### Opening an RFC
+
+1. Fork the lit/rfcs repo
+2. Draft the RFC
+    1. Add the RFC file to /rfcs in a markdown file named `NNNN-_descriptive-name_.md`
+        1. Note the `NNNN` is literal. RFCs will not have an assigned RFC number when first opened.
+3. Open an RFC pull request
+    2. Optionally share the draft on GH discussions or Slack before opening a pull request
+    3. Once opened, update the PR with a link to the PR in the metadata
```

Comment:
A link to which PR? The implementation? I wouldn't expect there to usually be a PR implementation already.

Reply:
The RFC PR so that you can get back to the discussion on it.

Reply:
Oh I see. Then I would say "update the RFC" instead of "update the PR".